### ISSUE TITLE
Menu Overlay: Navigation with Keyboard not working when first option is disabled

### ISSUE THREAD

**Issue by RohdeSchwarz-SDC** [primefaces/primefaces#14006]
### Describe the bug

When a simple overlay-menu has a disabled first option, then one cannot use arrow keys to navigate.

```
<p:commandButton value="Menu ▼" type="button"/>
<p:menu trigger="@previous" overlay="true">
	<p:menuitem value="Option 1" disabled="true"/>
	<p:menuitem value="Option 2"/>
	<p:menuitem value="Option 3"/>
	<p:menuitem value="Option 4"/>
</p:menu>
```

<img width="150" height="309" alt="Image" src="https://github.com/user-attachments/assets/ea22a423-fe3e-4068-9b46-fa1a9a15512e" />

### PrimeFaces edition

Community

### PrimeFaces version

14.0.12

### Theme

Saga

### JSF implementation

Mojarra

### JSF version

4.0.8

### Java version

24

### Browser(s)

all

**Comment by melloware** [comment]
i will look at this for 15.0.7
